flashinfer/jit/blackwell_bgmv_moe.py

```
@@ -0,0 +1,239 @@
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+"""JIT registration for the generated SM100 BGMV MoE programs."""
+
+from __future__ import annotations
```

Ruff (0.16.2)

[error] `from __future__` imports must occur at the beginning of the file

(F404)